src/main/java/frc/robot/Constants.java

```diff
@@ -34,12 +23,44 @@
 import frc.robot.subsystems.Elevator;
 
 public final class Constants {
+  public static class MotorIDConstants {
+    public static final int kCLimberMotorLeader = 12;
+    public static final int kCLimberMotorFollower = 13;
+    public static final int kScorerMotor = 9;
+    public static final int kElevatorMotor1 = 10;
+    public static final int kElevatorMotor2 = 11;
+    public static final int kIntakeMotor = 9;
+  }
+
+  public static class ClimberConstants {
+    public static final double kClimberP0 = 2;
+    public static final double kClimberI0 = 0;
+    public static final double kClimberD0 = 1;
+    public static final double kClimberkG0 = 0;
+    public static final double kClimberkV0 = 2;
+
+    public static final double kClimberP1 = 2;
+    public static final double kClimberI1 = 0;
+    public static final double kClimberD1 = 1;
+    public static final double kClimberkG1 = 0.5;
+    public static final double kClimberkV1 = 2;
+
+    public static final double KGearRatio = 126;
+    public static final Angle kClimberExtendedSetpoint = Degrees.of(190);
+    public static final Angle kClimberAtCageSetpoint = Degrees.of(170);
+    public static final Angle kClimberRetractedSetpoint = Degrees.of(90);
```

Comment:
Your code assumes that the encoders start up in this position, but the relative encoders will startup with position zero. 
You'll need to use setPosition() on the motors at the startup to tell them what their initial position is (remember to account for the gear ratio).

Check with the drivebase/climb mech team if absolute encoders are being put on the climber mechanism, because that would be the most accurate way to seed the initial position.